sdk/core/core/inc/az_http_header.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#ifndef AZ_HTTP_HEADER_H
+#define AZ_HTTP_HEADER_H
+
+#include <az_span_emitter.h>
+#include <az_pair.h>
+
+#include <_az_cfg_prefix.h>
+
+/**
+ * Emits an HTTP header as a sequence of spans in a format "%{key}: %{value}"
```

Comment:
is that replacement format literal, or does the %{} stand in for something.

Reply:
No, I just need some kind of syntax to explain what the resulted string.